packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -209,7 +209,7 @@ class _CupertinoTextSelectionControlsToolbar extends StatefulWidget {
     required this.textLineHeight,
   });
 
-  final ValueNotifier<ClipboardStatus>? clipboardStatus;
+  final ValueListenable<ClipboardStatus>? clipboardStatus;
```

Comment:
Why the changes from ValueNotifier to ValueListenable?

Reply:
It is not using the notifier api, the builder should not be mutating the state of ClipboardStatus anyway.